src/agentNet/agent_net.py

```diff
@@ -28,27 +35,55 @@ def create_network(self):
             self.network = nx.Graph(self.connections)
         else:
             self.network = nx.fast_gnp_random_graph(self.init_num, dens2prob(self.net_density))
-    def set_params(self, config_filename):
+    def set_policies(self, config_filename):
         """
-        Method sets randomly a file link to agent policy for each agent id. 
-        :param config_filename: path to file with links to policies
+        Method sets policies from config file. 
+        :param config_filename: path to file with policies config
         """
         with open(config_filename, 'r') as configs:
             params_yaml = yaml.load(configs)
-            self.agents_policies = {agent_id: random.choices(params_yaml.get('configs')) for agent_id in self.network.nodes}
-    def add_agent(self, agent_id):
+            flatten_values = list(itertools.chain.from_iterable(list(params_yaml['agents_policies'].values())))
```

Comment:
Czy wartości `params_yaml['agents_policies']` to kolekcje? Bo jeżeli klucze są identyfikatorami agentów to wartości powinny być politykami - zwłaszcza że zipujesz potem klucze i wartości, więc całość można by uprościć pewnie do:
```
 if not self.agents_policies:
    self.agent_policies = {}
 self.agents_policies.update(params_yaml['agents_policies'])
```